src/Authenticator/AuthenticatorCollection.php

```diff
@@ -58,6 +62,15 @@ public function __construct(IdentifierCollection $identifiers, array $config = [
     protected function _create(object|string $class, string $alias, array $config): AuthenticatorInterface
     {
         if (is_string($class)) {
+            if (!empty($config['identifier'])) {
+                $this->_identifiers = new IdentifierCollection($config['identifier']);
+            } else {
+                deprecationWarning(
+                    '3.3.0',
+                    'IdentifierCollection is deprecated. Directly pass `\'identifier\'` config to Authenticator.',
```

Comment:
```suggestion
                    'Having authenticators implicitly use an IdentifierCollection is deprecated. Directly pass `\'identifier\'` config to Authenticator.',
```

The current phrasing doesn't seem right as `IdentifierCollection` isn't being deprecated.

Reply:
Well the usage of it is (from the outside), no?
We are always directly assigning them

How would you phrase it?

Reply:
I rephrased it as:

> loadIdentifier() usage is deprecated. Directly pass Identifier to Authenticator.